Mobile-Workshop/Mobile-Workshop/Networking/NetworkManager.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  NetworkManager.swift
+//  Mobile-Workshop
+//
+//  Created by iOS Dev on 24.05.23.
+//
+
+import Foundation
+import Alamofire
+import Factory
+
+class NetworkManager {
+    private let session: Session
+    
+    init() {
+        let configuration = URLSessionConfiguration.default
+        configuration.timeoutIntervalForRequest = 30
+        self.session = Session(configuration: configuration, interceptor: JWTInterceptor())
```

Comment:
`JWTInterceptor` should be added as a module in Factory and be propagated via injection.